File: src/main/java/nz/co/searchwellington/model/taggingvotes/TaggingVote.java
package nz.co.searchwellington.model.taggingvotes;

import nz.co.searchwellington.model.Tag;
import nz.co.searchwellington.model.taggingvotes.voters.TaggingVoter;

public interface TaggingVote {
	
	public Tag getTag();
	public TaggingVoter getVoter();
	
}

File: src/main/scala/nz/co/searchwellington/model/taggingvotes/TaggingVote.scala
package nz.co.searchwellington.model.taggingvotes

import nz.co.searchwellington.model.{Tag, User}

trait TaggingVote {
  def tag: Tag
  def explanation: String
  def user: Option[User]

  def getTag: Tag = tag
  def getExplanation: String = explanation
  def getUser: User = user.orNull

}
